spp_change_request/models/mixins/source_mixin.py

```diff
@@ -711,6 +711,28 @@ def action_attach_documents(self):
                     _("There are no directories defined for this change request.")
                 )
 
+    def open_registrant_details_form(self):
```

Comment:
Add the documentation at the same time :D

Reply:
@jeremi noted on this.